src/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+      rel="stylesheet"
+    />
+    <title>Todo list</title>
+  </head>
+  <body>
+    <main>
+      <div class="title">
+        <p class="titl">Today's To Do</p>
```

Comment:
[Required Changes]There are similar CSS rule name which name "title" and "tit1" it's can be created different name